### ISSUE TITLE
Correct auth.resetPassword() API Reference

### ISSUE THREAD

**Issue by waleedshkt** [samuelgozi/firebase-auth-lite#29]
A small tweak to be made, mostly believed to be overlooked. 
In api reference docs, there is no second input argument for 'newPassord' mentioned.

**Comment by samuelgozi** [comment]
The reference API is outdated, I'll update it.
Thanks!

**Comment by waleedshkt** [comment]
@samuelgozi One more thing, by the way. What if to update the password when a user is signed in?

Should go for 'reset password' flow? But wouldn't it prompt the already authenticated user to get code from email?

Secondly and finally, in case of persisted logged in session i.e when user is automatically signed in on opening the app, how to get profile data (I see user data is returned when user is logged in or signed up)?

**Comment by samuelgozi** [comment]
I didn't understand the first question, can you rephrase?
As for the second question; You can access `auth.user`.

**Comment by waleedshkt** [comment]
No worries. I meant to say how to update the password for signed-in user?

**Comment by samuelgozi** [comment]
You can do that by passing the new password to `updateProfile` like this:
```js
auth.updateProfile({
  password: 'this is the new password'
})
```
